modules/tensor_mechanics/include/materials/czm/CZMEquilibriumTractionCalculatorBase.h

```diff
@@ -0,0 +1,44 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "InterfaceMaterial.h"
+/**
+ * Base class traction computing the traction used to impose equilibrium and its derivatives  w.r.t.
+ * the global displacement jump, starting from the values provided from any CZM constituive material
+ * model.
+ */
+class CZMEquilibriumTractionCalculatorBase : public InterfaceMaterial
```

Comment:
As I mentioned before, I would like to rename it `computeCZMGlobalTraction` or something similar. I just want to let users easily understand those are in local or global systems.